### PR TITLE
Add version examples

### PR DESCRIPTION
**Pull request by falsandtru**
Version examples(formats) are helpful.

### CODE REVIEW

File: issue_template.md [hunk 1]
```diff
@@ -13,11 +13,15 @@ __________________________________________________________ -->
 
 **TypeScript Version:** 
 
+1.7.5 / 1.8.0-beta / master(<date or hash>)
```
Comment (mhegazy) [inline issue_template.md]: i would change this to `1.7.5 / 1.8.0-beta / nightly (1.9.0-dev.20160217)`

**Comment by falsandtru** [comment]
Okay, I changed it.

**Comment by falsandtru** [comment]
I always test bug using git repository. How can I write this version?

**Comment by mhegazy** [comment]
no idea :D we can merge it and test i suppose.

**Comment by mhegazy** [comment]
thanks!

**Comment by falsandtru** [comment]
:+1:

**Pull request merged**